benchmark/bench.js

```diff
@@ -0,0 +1,60 @@
+const Client = require('../dist/index');
+const Benchmark = require('benchmark');
+const net = require('net');
+const suite = new Benchmark.Suite();
+
+(async function () {
+    const clientBuff = new Client.default({
+        host: 'udp://localhost:20000',
+        namespace: 'my.custom.namespace',
+        bufferSize: 1024,
+    });
+
+    const client = new Client.default({
+        host: 'udp://localhost:20000',
+        namespace: 'my.custom.namespace',
+    });
+
+    const server = net
+        .createServer((sock) => {
+            sock.on('data', (data) => undefined);
+        })
+        .listen(20001);
+
+    const clientTcp = new Client.default({
+        host: 'tcp://localhost:20001',
+        namespace: 'my.custom.namespace',
+        bufferSize: 1024,
+    });
+    await clientTcp.connect();
+
+    suite
```

Comment:
I would make the names more explicit, using udp and tcp in every one and using the same patterns. I think it would be easier to associate the bench result with the suite name in the graphs. At least for me 😄 

For example: `dats udp counter`, `dats tcp counter`, `dats udp counter buffered`, and so on.

Reply:
ok, It makes sense. I update it.